account/views.py

```diff
@@ -37,3 +41,18 @@ def logging_in(request):
 def logging_out(request):
     logout(request)
     return redirect(reverse('welcome'))
+
+
+def history(request):
+    for visit in Visit.objects.all():
+        out_of_date(visit)
```

Comment:
Let's discuss about it

Reply:
Copy-paste from another function. This piece of code is completely unnecessary here.